### PR TITLE
Add transifex-wg languages

### PR DESCRIPTION
**Pull request by igobranco**
Add Transifex Working Group languages.

New language codes: it_IT, pt_PT, de_DE, uk, ru, hi

@feanil 
@natabene

### CODE REVIEW

**Comment by natabene** [comment]
@igobranco Thank you for your contribution.

File: src/i18n/messages/hi.json [hunk 1]
```diff
@@ -0,0 +1,208 @@
+{
```
Comment (waheedahmed) [inline src/i18n/messages/hi.json:1]: Hello @igobranco!

You also need to import Hindi messages here https://github.com/openedx/frontend-app-authn/blob/4d1922c3b3617028a7832da66c82634427590f4f/src/i18n/index.jsx#L1-L33 and add the mapping into messages object.

**Review by waheedahmed — COMMENTED** [review]
Looks good except for one suggestion.

**Comment by waheedahmed** [comment]
@igobranco any update?

File: src/i18n/messages/hi.json (hunk 1 above)
Comment (igobranco) [reply src/i18n/messages/hi.json:1]: Ok.
I've included also the new languages that is the objective of this PR.

File: src/i18n/index.jsx [hunk 2]
```diff
@@ -4,6 +4,11 @@ import caMessages from './messages/ca.json';
 import es419Messages from './messages/es_419.json';
 import frMessages from './messages/fr.json';
 import zhcnMessages from './messages/zh_CN.json';
+import ititMessages from './messages/it_IT.json';
+import ptptMessages from './messages/pt_PT.json';
+import ptptMessages from './messages/pt_PT.json';
```
Comment (waheedahmed) [inline src/i18n/index.jsx:9]: Please remove this duplicate import.

**Review by waheedahmed — CHANGES_REQUESTED** [review]

**Review by waheedahmed — APPROVED** [review]

**Comment by edx-webhook** [comment]
@igobranco 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-authn&entry.867055334=https://github.com/openedx/frontend-app-authn/pull/463&entry.1484655318=https://github.com/igobranco&entry.752974735=2021-11-08+13:05&entry.1917517419=2022-03-29+12:04&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**